source/_components/notify.group.markdown

```diff
@@ -48,3 +48,14 @@ services:
       required: false
       type: string
 {% endconfiguration %}
+
+## {% linkable_title Example %}
+
+An example on how to use inside of an automation:
+
+```yaml
+action:
+  service: notify.NAME_OF_NOTIFIER_GROUP
+  data:
+    message: "The sun is {% raw %}{% if is_state('sun.sun', 'above_horizon') %}up{% else %}down{% endif %}{% endraw %}!"
```

Comment:
`{% raw %}` should be wrapped outside the code block right (outside of the opening of the yaml code block)?
Since the user is not supposed to copy that part I assume.

Reply:
No, the code is the same as the example on the other page: [Edit notify-markdown](https://github.com/home-assistant/home-assistant.io/edit/current/source/_components/notify.markdown).
I assume it should be the same as there?!

Reply:
Well, that is incorrect as well in that case...
The raw now causes the parts between it not to be executed in the user's example...

Reply:
Sorry, forgot this! :)
Sounds more like I should remove the raw-tags as the example should be copy&paste-enabled